docs/source/features/reasoning_outputs.md

```diff
@@ -172,8 +177,8 @@ print("content: ", completion.choices[0].message.content)
 
 ## Limitations
 
-- The reasoning content is only available for online serving's chat completion endpoint (`/v1/chat/completions`).
-- It is not compatible with [`tool_calling`](#tool_calling).
+* The reasoning content is only available for online serving's chat completion endpoint (`/v1/chat/completions`).
+* It is not compatible with [`tool_calling`](#tool_calling).
```

Comment:
Can we rever this to reduce code change? thanks.

Reply:
Sure, done!